plugins/auth-keycloak/vite.config.ts

```diff
@@ -0,0 +1,21 @@
+import { defineConfig } from 'vite';
+import react from '@vitejs/plugin-react-swc';
+
+// https://vitejs.dev/config/
+export default defineConfig(({ mode }) => ({
+  build: {
+    minify: false,
+    lib: {
+      entry: './src/index.ts',
+      fileName: () => 'index.js',
+      formats: ['cjs'],
+    },
+    rollupOptions: {
+      external: ['react', 'react-dom', 'redux', 'react-redux'],
+      // test2
```

Comment:
Oops I put this in the plotly-express Vite config when I was testing something w/ the watch mode. Can you remove it in both places?